src/bibliography.jl

```diff
@@ -1,9 +1,20 @@
-
 abstract type BibliographyBlock <: Expanders.ExpanderPipeline end
 
 Selectors.order(::Type{BibliographyBlock}) = 12.0  # Expand bibliography last
 Selectors.matcher(::Type{BibliographyBlock}, node, page, doc) = Expanders.iscode(node, r"^@bibliography")
 
+const tex2unicode_replacements = (
+    "--"  => "–",
+    "---" => "—"
+)
+
+function tex2unicode(s)
+    for replacement in tex2unicode_replacements
+        s = replace(s, replacement)
+    end
+    return s
```

Comment:
I think you can use multiple pairs in one `replace`
```suggestion
        replace(s, tex2unicode_replacements...)
```

Reply:
Woah nice!

Reply:
Ah looks like

```julia
replace(A, old_new::Pair...; [count::Integer])
```

is defined in general but for strings we are stuck with

```julia
replace(s::AbstractString, pat=>r; [count::Integer])
```

:(

Reply:
Ah, ok.

Reply:
Ah, I just realised. Upstream issue https://github.com/JuliaLang/julia/issues/35327